pkg/config/constants/constants.go

```diff
@@ -163,7 +163,6 @@ const (
 	WaypointServiceAccount = "istio.io/for-service-account"
 
 	ManagedGatewayLabel               = "gateway.istio.io/managed"
-	ManagedGatewayController          = "istio.io/gateway-controller"
```

Comment:
I think this constant should stay and be used as the default for the env variable